src/content/teams/13_geospatial.mdx

```diff
@@ -1,16 +1,17 @@
 ---
 name: Geospatial Team
-description: Making Nix and NixOS the first choice for users of geospatial software
+description: Maintains geospatial software and related NixOS modules.
 members:
-  - name: Ivan Mincik
-    discourse: imincik
+  - name: Will Cohen
+    discourse: willcohen
```

Comment:
Small edit — discourse username is wcohen

Reply:
Good point, thank you. Added both github and discourse names.